distributions/openhab/src/main/resources/bin/update

```diff
@@ -95,11 +95,11 @@ setup(){
     echo "You are already on openHAB $CurrentVersion" >&2
     exit 1
   elif [ -n "$milestoneVersion" ]; then
-    DownloadLocation="https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab/$OHVersion/openhab-$OHVersion.zip"
-    AddonsDownloadLocation="https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab-addons/$OHVersion/openhab-addons-$OHVersion.kar"
+    DownloadLocation="https://www.openhab.org/download/milestones/org/openhab/distro/openhab/$OHVersion/openhab-$OHVersion.zip"
+    AddonsDownloadLocation="https://www.openhab.org/download/milestones/org/openhab/distro/openhab-addons/$OHVersion/openhab-addons-$OHVersion.kar"
   else
-    DownloadLocation="https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab%2F$OHVersion%2Fopenhab-$OHVersion.zip"
-    AddonsDownloadLocation="https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab-addons%2F$OHVersion%2Fopenhab-addons-$OHVersion.kar"
+    DownloadLocation="https://www.openhab.org/download/releases/org/openhab/distro/openhab/$OHVersion/openhab-$OHVersion.zip"
+    AddonsDownloadLocation="https://www.openhab.org/download/releases/org/openhab/distro/openhab-addons/$OHVersion/openhab-addons-$OHVersion.kar"
```

Comment:
It gives a 404 with this URL:

https://www.openhab.org/download/releases/org/openhab/distro/openhab/3.0.1/openhab-3.0.1.zip

There is no content in:

https://openhab.jfrog.io/artifactory/libs-release-local/

Reply:
Yes, because so far no releases have been published there. We will need to do a 3.0.2 patch release in April, before Bintray shuts down our account.

Reply:
Will previous releases also be available there before Bintray sunsets?
Some users might like to download them after February 1st, 2022.

Reply:
> We will need to do a 3.0.2 patch release in April, before Bintray shuts down our account.

I would also need to update the [URL](https://github.com/openhab/openhab-docker/blob/4b021c7f6ca71ee4e1a1c50b344a3945d3325796/debian/Dockerfile#L93) for the Docker build for this. So I'd have to keep track of multiple download URLs to also support building OH <= 3.0.1 containers. Something similar may need to be done for .deb/.rpms by @BClark09 and others.

Reply:
@wborn Let us better discuss this on https://github.com/openhab/openhab-distro/issues/1256.
This change here is merely about the update script as this is rolled out to the users and hence we have to make sure that all snapshot/milestone/release users that do manual upgrades through this script have the new urls on their local installation already before Bintray is shut down. For this case, I'd think it is ok to only support "future" releases as the script is primarily meant for upgrades, not for downgrades.

Reply:
OK then let's merge this but please answer my questions in #1256. The checklist definitely also needs updating download URLs in all repos that package openHAB.